logging/google/cloud/logging/logger.py

```diff
@@ -123,19 +128,24 @@ def _make_entry_resource(self, text=None, info=None, message=None,
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (Optional) timestamp of event being logged.
 
+        :type resource :class:``google.cloud.logging.resource`
```

Comment:
This reference should be ":class:\`google.cloud.logging.resource.Resource\`". (seems like this needs to be fixed in a few places)